### DISCUSSION TITLE
Confused about naming

### DISCUSSION THREAD

**Discussion by TheRisenPhoenix** [online-ml/river#520] [Q&A] [answered]
I'm new to river and a bit confused about the naming.
When I use
`pip install git+https://github.com/online-ml/river --upgrade`
(as recommended by the documentation, for installing the latest development version), I get river@0.1.0.

However, looking at the releases, the current version seems to be 0.6.1.
So, I tried installing using this command:
`pip install git+git://github.com/online-ml/river.git@0.6.1`
But now, the package in my project is "creme". However, both seem to have the same functionalities (at first glance).

I'm a bit confused now, which is the recommended way to install the latest version? Do creme and river provide the same functionalities?
Unfortunately, the docs seem a bit outdated, and also in parts broken (many links in the descriptions don't work) 😕

**Comment by raphaelsty** [comment, marked as answer]
Hi @TheRisenPhoenix,

The command `pip install git+https://github.com/online-ml/river --upgrade` download the latest development version.

The latest __version__ file `https://github.com/online-ml/river/blob/master/river/__version__.py` is set to 0.1.0, so everything is ok having river@0.1.0.

Creme and Scikit-multiflow have joined together to create a single library River. The second command you used downloaded Creme because its source code was hosted on this repository. Creme will not be updated anymore. I recommend you to use River, the library is maintained, offers new features and the documentation is more recent.

**Reply by TheRisenPhoenix** [reply]
Ah okay! Thanks for clarifying this for me! 🙂
